coruscant/src/components/RegisterPrompt.js

```diff
@@ -0,0 +1,123 @@
+import React, {Component} from 'react';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import AppBar from 'material-ui/AppBar';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import * as validator from '../utils/validator';
+
+class Register extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            email: '',
+            username: '',
+            password: '',
+            success: false
+        }
+    }
+
+    checkUsername(newValue) {
+        this.setState({username: newValue});
+        var error = validator.validateUsername(newValue);
+        if (error === true) {
+            this.setState({usernameError: ""});
+            return true;
+        } else {
+            this.setState({usernameError: error});
+            return false;
+        }
+    }
+
+    checkEmail(newValue) {
+        this.setState({email: newValue});
+        var error = validator.validateEmail(newValue);
+        if (error === true) {
+            this.setState({emailError: ""});
+            return true;
+        } else {
+            this.setState({emailError: error});
+            return false;
+        }
+    }
+
+    checkPassword(newValue) {
+        this.setState({password: newValue});
+        var error = validator.validatePassword(newValue);
+        if (error === true) {
+            this.setState({passwordError: ""});
+            return true;
+        } else {
+            this.setState({passwordError: error});
+            return false;
+        }
+    }
+
+    handleClick(event) {
+        var self = this;
+        var validEmail = this.checkEmail(this.state.email);
+        var validPassword = this.checkPassword(this.state.password);
+        var validUsername = this.checkUsername(this.state.username);
+        if (validEmail && validPassword && validUsername) {
+            console.log({
+                "email": this.state.email,
+                "username": this.state.username,
+                "password": this.state.password
```

Comment:
Remove the password being logged to the console. Just so there's no chance it gets left in later.